cmd/operator-sdk/internal/genutil/k8s.go

```diff
@@ -34,6 +35,16 @@ import (
 func K8sCodegen() error {
```

Comment:
Also, just one question, is the issue just faced with the operator-sdk generate k8s command?

Reply:
Good question. I tested in a failing environment (where I have release `v0.16.0` and unset `GOROOT`), and both the `crds` and `csv` generate commands work without throwing any errors.

This seems to be isolated to the `k8s` generation.